config/manager-base/manager_config_patch.yaml

```diff
@@ -10,6 +10,7 @@ spec:
       - name: manager
         args:
         - "--config=controller_config.yaml"
+        - "--user-config-name=kmm-user-override-config"
```

Comment:
I think having 2 configMaps to configure KMM can be confusing for customers. Especaially for the one that already uses `controller_config` to configure KMM as mentioned in the docs.

How about we set all the default in the code (at the upper layer as possible) and just extract `controller_config` configMap from the bundle. The benefits are:
* User will still use the same configMap as before
* It will persist during upgrades
* Only small documentation changes are required
* We don't have 2 configMap to do the same (and only one of them is persistent which is even worst)